package.json

```diff
@@ -19,7 +20,8 @@
     "danger": "danger",
     "fix": "eslint . --fix",
     "promote": "heroku pipelines:promote -r staging",
-    "dump-schema": "babel-node ./scripts/dump-schema.js"
+    "dump-schema": "babel-node ./scripts/dump-schema.js",
+    "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema"
```

Comment:
😮  quite the script. I hope this doesn't trip us up if we decide to add new folders to this project.